torch/csrc/distributed/c10d/init.cpp

```diff
@@ -3203,6 +3203,23 @@ such as `dist.all_reduce(tensor, async_op=True)`.
         Arguments:
           tensors(List[torch.Tensor]): List of tensors we want to hash.
       )");
+  module.def(
+      "_dump_nccl_trace_json",
+      [](std::optional<bool> includeCollectives,
+         std::optional<bool> onlyActive) {
+        return py::bytes(::c10d::dump_nccl_trace_json(
```

Comment:
Should we be releasing the GIL during dumping?

Reply:
Unsure about this.